modules/oe_content_event/src/Plugin/InternalLinkSourceFilter/EventPeriodFilter.php

```diff
@@ -33,6 +33,11 @@
  */
 class EventPeriodFilter extends InternalLinkSourceFilterPluginBase implements InternalLinkSourceFilterInterface, ContainerFactoryPluginInterface {
 
+  /**
+   * Option for ongoing events.
+   */
+  const ONGOING = 2;
```

Comment:
Maybe even nicer to use 0 integer value for ongoing event constant (as it is a moment between past and upcoming).

Reply:
You can't use 0 there, it will considered as empty and it will show all events.